Projects/SEC/SEC04/SEC04-AWS100.md

```diff
@@ -0,0 +1,60 @@
+
+# SEC04-AWS100 — Create an IAM user
+
+## Cloud Service Provider
+
+* Amazon Web Services
+
+## Difficulty
+* Level 100 (Introductory)
+
+## Estimated time:
+ * 1 hour
+
+## Cost
+* IAM is a free service
+
+## Project's Author(s)
+* [Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+
+* Create a new user in AWS IAM with programmatic and console access
+* Add user to Admin group 
+--Create a new group called "Admins"
+--Add the AdministratorAccess managed policy to the new group
+--Add your new user to the "Admins" group
```

Comment:
These subheadings are not working Markdown.
You can do subheadings with indentation:

```
* Add user to an Admin group
  * Create a new group called "Admins"
  * Add the `AdministratorAccess` managed policy to the new group
  * Add your new user to the "Admins" group
```
⬇️
* Add user to an Admin group
  * Create a new group called "Admins"
  * Add the `AdministratorAccess` managed policy to the new group
  * Add your new user to the "Admins" group